modules/API/pages/built-in-endpoints.adoc

```diff
@@ -1070,6 +1070,55 @@ Response::
 
 None.
 
+=== Remove expired tokens
+
+`DELETE :8123/expiredtoken`
+
+This endpoint removes expired authentication tokens.
+A user is always allowed to remove their own expired tokens.
+If a user tries to delete tokens that belong to other users, they need to have xref:user-access:access-control-model.adoc#_privileges[the `WRITE_USER` privilege].
+
+==== Sample request
+
+The following request deletes all expired tokens.
+This request requires the privilege `WRITE_USER`:
+
+[.wrap,console]
+----
+curl -X DELETE "https://localhost:8123/expiredtoken"
+----
+
+The following request deletes all expired tokens that belong to users `u1` and `u2` as well as all tokens created with secrets `s1` and `s2`.
+This request requires the privilege `WRITE_USER`
+
+[.wrap,console]
+----
+curl -X DELETE 'localhost:8123/expiredtoken' -d '{"user":["u1","u2"],"secret":["s1","s2"]}'
+----
+
+==== Request body
+The request body is optional.
+If not provided a request body, a request deletes all expired tokens.
+
+[.wrap,javascript]
+----
+{
+    "user": ["string"], <1>
+    "secret": ["string"] <2>
+}
+----
+<1> Users whose expired tokens to remove. Optional.
+<2> If a token is created with one of the secrets in the list, they are removed by the request. Optional.
+
+The secrets and users provided in the request body do not have to correlate with each other.
+A request deletes all tokens associated with any of the users and secrets contained in the payload.
```

Comment:
It might worth to mention that this endpoint behavior is atomic and illustrate what happens if any of payload parameter is non-exist, invalid or the user does not have sufficient privilege.
Essentially, if any user or secret in the payload is invalid/non-exist, or the current user does not have enough privilege on some user/secret, then the request will fail and none of the expired tokens will be deleted.

Reply:
Added the clarification